src/commands/cancel.ts

```diff
@@ -20,6 +22,18 @@ export const cancel: Command = {
 
     services.scheduler.cancel(name, message.guild.name);
     //remove persistance once implemented
+    const jobs = await services.store.get<StorableJob[]>("jobs");
+    if (!jobs) {
+      throw new Error("Failed to get jobs from store");
+    }
+
+    const index = jobs.findIndex((job) => job.name === name && job.message.guild === guildName);
+
+    if (index === -1) {
+      throw new Error("Job not found in store array");
```

Comment:
I don't think this needs to be an explicit error. It's not an unexpected state to be in, so we should return a reply saying "No x job to remove"